performance_regression_gradual_grow_throughput.py

```diff
@@ -186,30 +232,14 @@ def run_gradual_increase_load(self, stress_cmd_templ, stress_num, num_loaders, c
         with open(filename, "w", encoding="utf-8") as res_file:
             res_file.write(total_summary_json)
 
-        screenshots = self.monitors.get_grafana_screenshots(self.monitors.nodes[0], self.start_time)
```

Comment:
we don't generate the overwall screenshots anymore ? 

can you clarify why you are remove this this ?

Reply:
It is part of `self.update_test_details(scylla_conf=True)` that called in each step